docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  task_db:
+    image: postgres
+  task:
+    build: .
+    command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8001"
```

Comment:
@ahmed had some thoughts about alternative deployment strategies. Tagging him here so that he's aware. Altering deployment is probably a second PR.